lib/auth/cognito_chooser.rb

```diff
@@ -24,20 +27,30 @@ def aws_secret_key
     Rails.configuration.cognito_aws_secret_access_key
   end
 
+  def region
+    Rails.configuration.aws_region
+  end
+
+  def user_pool_id
```

Comment:
do we need any of these? (region and user_pool_id)

Reply:
We need region not the user_pool_id which I have no removed

Reply:
where is the region used? I see the `register_dev_client` using`Rails.configuration.aws_region` directly